tests/test_setup_marathon_job.py

```diff
@@ -54,7 +54,7 @@ class TestSetupMarathonJob:
         'password': 'admin_pass',
     }, '/fake/fake_file.json')
     fake_args = mock.MagicMock(
-        service_instance='what_is_love.bby_dont_hurt_me',
+        service_instance_list=['what_is_love.bby_dont_hurt_me'],
```

Comment:
args can't really be an array, they will be a string, you are going to have to csv this or do multiple args